theme_kiddo/views/images_library.xml

```diff
@@ -107,36 +142,10 @@
     <field name="name">website.s_company_team_image_4</field>
     <field name="url">/theme_kiddo/static/src/img/content/content_img_29.jpg</field>
 </record>
-
-<record id="s_product_list_default_image_1" model="theme.ir.attachment">
-    <field name="key">website.s_product_list_default_image_1</field>
-    <field name="name">website.s_product_list_default_image_1</field>
-    <field name="url">/theme_kiddo/static/src/img/snippets/s_product_list_1.svg</field>
-</record>
-<record id="s_product_list_default_image_2" model="theme.ir.attachment">
-    <field name="key">website.s_product_list_default_image_2</field>
-    <field name="name">website.s_product_list_default_image_2</field>
-    <field name="url">/theme_kiddo/static/src/img/snippets/s_product_list_2.svg</field>
-</record>
-<record id="s_product_list_default_image_3" model="theme.ir.attachment">
-    <field name="key">website.s_product_list_default_image_3</field>
-    <field name="name">website.s_product_list_default_image_3</field>
-    <field name="url">/theme_kiddo/static/src/img/snippets/s_product_list_3.svg</field>
-</record>
-<record id="s_product_list_default_image_4" model="theme.ir.attachment">
-    <field name="key">website.s_product_list_default_image_4</field>
-    <field name="name">website.s_product_list_default_image_4</field>
-    <field name="url">/theme_kiddo/static/src/img/snippets/s_product_list_4.svg</field>
-</record>
-<record id="s_product_list_default_image_5" model="theme.ir.attachment">
-    <field name="key">website.s_product_list_default_image_5</field>
-    <field name="name">website.s_product_list_default_image_5</field>
-    <field name="url">/theme_kiddo/static/src/img/snippets/s_product_list_5.svg</field>
-</record>
-<record id="s_product_list_default_image_6" model="theme.ir.attachment">
-    <field name="key">website.s_product_list_default_image_6</field>
-    <field name="name">website.s_product_list_default_image_6</field>
-    <field name="url">/theme_kiddo/static/src/img/snippets/s_product_list_6.svg</field>
```

Comment:
As always we say it is ok to change all the images of the themes despite the fact that people who kept the themes ones will see their website getting new images at some places (I personally find that not perfect but still ok as people are likely to change the default images and mainly: the problem is now only when they did not use the configurator).

However for these 6 specific SVG ones, the problem is the same with configurator or not and those are images that are more likely to be kept by the user. After migration, the user gets this result (at best):
![image](https://user-images.githubusercontent.com/10338094/144877737-6b7946b9-199d-4f6d-999e-74849462e98d.png)

Maybe for these 6 ones we should keep the ir.attachment records and you would just need to update the URLs (and add a comment to say it is for compatibility)?